src/objective-c/ProtoRPC/ProtoMethod.m

```diff
@@ -46,7 +46,7 @@ - (instancetype)initWithPackage:(NSString *)package
 }
 
 - (NSString *)HTTPPath {
-  if (_package) {
+  if (_package && _package.length > 0) {
```

Comment:
nit: `if (_package.length > 0) {` achieves the same, and IIRC it's how it's commonly found in the codebase. I don't have a strong preference one way or the other anyway.